### ISSUE TITLE
Implement Session Storage for search input field 

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#11334]
@roseeichelmann brought this up in Dev Sync 1/26/23 as an issue that she discovered while building out the FTT Fatalities Index Page, which is using the same search pattern.

----

As a staff member, I want the ability to clear the search input after every session so that previous advanced search criteria is not referenced and I can start fresh with new search criteria, in order to mitigate confusion and ensure accurate search results.

Acceptance criteria for moving from local storage to session storage:

- The search input field is automatically reset to its default state after a user closes their browser.
- Advanced search criteria is not referenced when a user re-opens the browser and uses the search input field.
- A clear button is still present, but it will now be used to clear the search input field within the current session as well.
- The user will be notified of the change in the storage method, and provided with instructions on how to use it.

**Comment by patrickm02L** [comment]
### Update
Patrick experienced this Issue first hand when logging into VZE after a long weekend and thought that there was a bug effecting the display of the Index. With the help of the Devs he was informed that he needed to clear the filters to get the records to show. 

It was recommended that we might think about changing the UI patter to make it more obvious that there is a filter applied. 

![Screen Shot 2023-07-25 at 8 31 44 AM](https://github.com/cityofaustin/atd-data-tech/assets/5566727/2affd9b2-cc93-465a-acf9-99ca5624fe11)

**Comment by johnclary** [comment]
Yep—this is super confusing and i think it would be best handled by saving search statate using query params in the URL. This is what we did in Moped and we're happy with it. I'm renaming this issue 👍
